p2p/src/authenticated/actors/listener.rs

```diff
@@ -24,9 +24,11 @@ pub struct Config<C: Scheme> {
     pub allowed_incoming_connection_rate: Quota,
 }
 
+type Sink<E> = <<E as Network>::Listener as Listener>::Sink;
```

Comment:
I suppose there isn't a way to reuse this in `dialer`?

Reply:
Added `StreamOf` and `SinkOf` types to make this cleaner across the code